lib/graphql/account_settings/cloud_object.dart

```diff
@@ -0,0 +1,146 @@
+import 'dart:async';
+
+import 'package:autonomy_flutter/common/injector.dart';
+import 'package:autonomy_flutter/database/cloud_database.dart';
+import 'package:autonomy_flutter/gateway/iap_api.dart';
+import 'package:autonomy_flutter/graphql/account_settings/account_settings_client.dart';
+import 'package:autonomy_flutter/graphql/account_settings/account_settings_db.dart';
+import 'package:autonomy_flutter/graphql/account_settings/cloud_object/address_cloud_object.dart';
+import 'package:autonomy_flutter/graphql/account_settings/cloud_object/connection_cloud_object.dart';
+import 'package:autonomy_flutter/service/configuration_service.dart';
+import 'package:autonomy_flutter/service/settings_data_service.dart';
+import 'package:autonomy_flutter/util/migration/migration_util.dart';
+import 'package:package_info_plus/package_info_plus.dart';
+
+class CloudObjects {
```

Comment:
the class name look like a `model`, but actualy, it's `util`

Reply:
manager is good